examples/controls_example/public/react_controls/control_group/get_control_group_factory.tsx

```diff
@@ -167,6 +169,19 @@ export const getControlGroupEmbeddableFactory = (services: {
         dataViews.next(newDataViews)
       );
 
+      /** Fetch the allowExpensiveQuries setting for the children to use if necessary */
+      try {
+        const result = await services.core.http.get<{
+          allowExpensiveQueries: boolean;
+          // TODO: Rename this route as part of https://github.com/elastic/kibana/issues/174961
+        }>('/internal/controls/optionsList/getExpensiveQueriesSetting', {
+          version: '1',
+        });
+        allowExpensiveQueries$.next(result.allowExpensiveQueries);
+      } catch {
+        allowExpensiveQueries$.next(true); // default to true on error
+      }
```

Comment:
This **used** to be fetched per options list control - however, I decided to move this into the control group to simplify the fetching.